test/testUnits.cc

```diff
@@ -184,14 +190,21 @@ SUITE(Units)
     from2->setUnits(from1->unitsStr());
     CHECK_THROW(opp->checkUnits(),std::exception);
   }
+
+  template <> void TestOp::impl<OperationType::polygamma>()
+  {
+    init(OperationType::polygamma);
+    // log's arguments must be dimensionless
```

Comment:
Please update comment.

Reply:
Thanks for catching this, I've updated it.